lib/kv/dummy.js

```diff
@@ -63,6 +68,27 @@ var DummyKvResource = DummyResource.extend(function(self, name, store) {
         return value;
     };
 
+    self.expire = function(key, seconds) {
+        /**:DummyKvResource.expire(key[, seconds])
+
+        Set or remove the expiry of a key.
+
+        If seconds is ``null``, the key is set not to expire.
```

Comment:
On second thought, maybe that is a bit confusing. If this is called `expire`, then using it as `.expire('foo')` to set a key to not expire maybe isn't clear enough. `.expire('foo', null)` seems quite clear. Maybe we should just unexpire on null, and assume that any other value that is given is a number.